distro/src/bin/hz

```diff
@@ -20,19 +20,75 @@ PRGDIR=`cd "$PRGDIR"; pwd`
 # set the parent directory as HAZELCAST_HOME.
 export HAZELCAST_HOME=${HAZELCAST_HOME:-$(cd "$PRGDIR/.." >/dev/null; pwd -P)}
 
-if [ $JAVA_HOME ] ; then
-    RUN_JAVA=$JAVA_HOME/bin/java
-else
-    RUN_JAVA=`command -v java 2>/dev/null`
+# common.sh needs it
+SCRIPT_DIR="$PRGDIR"
+
+. "$PRGDIR"/common.sh
+
+
+if [ "${PROMETHEUS_PORT}" ]; then
+    echo "Prometheus enabled on port ${PROMETHEUS_PORT}"
+    PROMETHEUS="-javaagent:${HAZELCAST_HOME}/lib/jmx_prometheus_javaagent-0.14.0.jar=${PROMETHEUS_PORT}:${HAZELCAST_HOME}/config/jmx_agent_config.yaml"
 fi
 
-if [ -z $RUN_JAVA ] ; then
-    echo "Error: JRE not found. Please install JRE 8 or higher in your PATH or set JAVA_HOME appropriately"
-    exit 1
+readJvmOptionsFile "jvm.options"
+
+if [ "$HAZELCAST_LICENSE_KEY" ]; then
```

Comment:
We always use the OS version with the CLI, right? Or - what is the license key used for?

Reply:
we have the same on `hz-start` and hmm, why won't the CLI be used with EE?

Reply:
I guess you're right, it can be used, although not sure how someone could install the EE version of the hazelcast JAR. Anyway, not a big deal.

Reply:
Yeah, I am also not sure :/ we keen keep it for now probably, just in case ...